src/app/page.tsx

```diff
@@ -26,8 +31,20 @@ export default async function HomePage() {
           </div>
 
           <div className="mt-6 grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3">
-            {new Array(6).fill(true).map((_, idx) => {
-              return <ArticlePreview key={idx} />;
+            {articles.map(article => {
+              return (
+                <ArticlePreview
+                  key={article.id}
+                  id={article.id}
+                  title={article.title}
+                  description={article.description}
+                  cover={article.cover}
+                  author={{
+                    name: article.author.username,
+                    avatar: article.author.avatar,
+                  }}
+                />
```

Comment:
Should we pass the article as props instead of each field as props?

Reply:
✅ Done